internal/cloud/gcp/compute.go

```diff
@@ -288,3 +315,108 @@ func (g *GCP) ComputeDiskDelete(ctx context.Context, zone, disk string) error {
 
 	return err
 }
+
+// storageRegionToComputeRegion translates a Google Storage Region to GCE Region.
+// This is useful mainly for multi and dual Storage Regions. For each valid multi
+// or dual Region name, a slice with relevant GCE Regions is returned. If the
+// Region provided as an argument is not multi or dual Region, a slice with the
+// provided argument as the only item is returned.
+//
+// In general, Storage Regions correspond to the Compute Engine Regions. However,
+// Storage allows also Multi and Dual regions, which must be mapped to GCE Regions,
+// since these can not be used with GCE API calls.
+//
+// Uses:
+//  - Compute Engine API
+func (g *GCP) storageRegionToComputeRegions(ctx context.Context, region string) ([]string, error) {
+	regionLower := strings.ToLower(region)
+
+	// Handle Dual-Regions
+	// https://cloud.google.com/storage/docs/locations#location-dr
+	if regionLower == "asia1" {
```

Comment:
Personally I have doubts that it is a good idea to hardcode the list of zones in the code. Mainly because it will be impossible to change this e.g. when the osbuild-composer version containing the change will end up in a RHEL EUS release and Google decides to change the definition of any of the Dual-Regions. I would be curious what others think. I would be more happy with making this dynamic. Maybe return the same list of Regions for multi-region as for the Dual regions, as this is mostly about where are Google's daisy workflow creates temporary resources for the image import. So one could argue that as long it is not on the other continent, it should not be that big of an issue, but I don't have any data to support this.

Reply:
Isn't there an API?

Reply:
Unfortunately I was not able to find any relevant API.

There is an API for listing all Compute Engine regions and zones within each region and I'm using this API in the code. However the "dual" and "multi" Cloud Storage regions are defined on top of the GCE Regions, in addition to GCE Regions. The GCE API knows about its regions and the Storage API does not allow one to list regions at all.

Reply:
Ah, that sucks, thanks for checking! :)